### DISCUSSION TITLE
Getting started doc doesn't compile out of the box

### DISCUSSION THREAD

**Discussion by KristaRutz** [open-telemetry/opentelemetry-dotnet#2954] [Q&A] [answered]
Specifically, the AddConsoleExporter extension can't be found. I have tried this in my DI helper project (.NET Standard 2.0), a console project (.NET Framework 4.8) and finally made a brand new console project with the basic `dotnet new console --output sample1` boilerplate (.NET 6.0). Each project references OpenTelemetry.Exporter.Console (v1.1.0, I also tried 1.0.1).  Each time, I get this failure:

`CS1061: 'MeterProviderBuilder' does not contain a definition for 'AddConsoleExporter' and no accessible extension method 'AddConsoleExporter' accepting a first argument of type 'MeterProviderBuilder' could be found (are you missing a using directive or an assembly reference?).`

I have followed the code here exactly. Copy/paste. Have the same three using statements.
https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/docs/metrics/getting-started/Program.cs
https://github.com/open-telemetry/opentelemetry-dotnet/blob/69817953f1487280f0308a60b9435202d3222e9c/docs/metrics/getting-started/Program.cs#L30

I see in this file the extension method that I need, so why can't I access it? https://github.com/open-telemetry/opentelemetry-dotnet/blob/69817953f1487280f0308a60b9435202d3222e9c/src/OpenTelemetry.Exporter.Console/ConsoleExporterMetricsExtensions.cs#L33

**Comment by cijothomas** [comment, marked as answer]
Metrics support is only for 1.2.* versions. It looks like you are using 1.1.0? Could you update to 1.2.* and retry

**Reply by KristaRutz** [reply]
The latest version I can see is 1.1.0 - https://www.nuget.org/packages/OpenTelemetry.Exporter.Console/1.1.0
Am I missing something?

**Reply by KristaRutz** [reply]
Seeing that it is in pre-release. Will try that, thanks. In the meantime, is there an ETA on when the release will be generally available?

**Reply by Yun-Ting** [reply]
This command will install the 1.2.0-rc2 version for you:
`
dotnet add package --prerelease OpenTelemetry.Exporter.Console
`
